immuni_exposure_ingestion/apis/ingestion.py

```diff
@@ -200,23 +179,32 @@ async def upload(  # pylint: disable=too-many-arguments
 @slow_down_request
 @cache(no_store=True)
 @monitor_check_otp
-async def check_otp(request: Request, is_dummy: bool, padding: str) -> HTTPResponse:
+@handle_dummy_requests(
+    [
+        WeightedPayload(
+            config.DUMMY_DATA_TOKEN_ERROR_CHANCE_PERCENT,
```

Comment:
I know I'm always asking for renamings, 😅 but what do you think about `DUMMY_DATA_TOKEN_ERROR_RATIO`?

Reply:
No worries about the renamings, I think naming is one of the hardest things to do, so it's good to pick names that are more clear for everyone.

That said, I prefer the current name `DUMMY_DATA_TOKEN_ERROR_CHANCE_PERCENT` over `DUMMY_DATA_TOKEN_ERROR_RATIO` because it's not a ratio, it's a percentage, so it also informs the reader that:
- The maximum value should be 100
- The opposite value is 100 - <value>

if we were instead to rename the error (and use a ratio) we should also define the precision, which is another issue 😞

Reply:
![image](https://user-images.githubusercontent.com/17234641/85044840-7aaf4280-b18e-11ea-95f8-bd2ae57a1e07.png)